massa-async-pool/Cargo.toml

```diff
@@ -19,7 +19,7 @@ massa_serialization = { path = "../massa-serialization" }
 massa_signature = { path = "../massa-signature" }
 massa_db_exports = { path = "../massa-db-exports" }
 massa_time = { path = "../massa-time" }
-massa-proto-rs = { git = "https://github.com/massalabs/massa-proto-rs", rev = "18ec02f", features = ["tonic"] }
+massa-proto-rs = { git = "https://github.com/massalabs/massa-proto-rs", branch = "feature/Improve_ABI_types_in_wasmv1", features = ["tonic"] }
```

Comment:
if you want to merge into testnet_24
you must point to a commit sha, not to a branch id that can change